utils/docker-ce/Makefile

```diff
@@ -0,0 +1,99 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=docker-ce
+PKG_VERSION:=18.09.6
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/docker/docker-ce/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=e8d2dd41e09e838e9043d4a5cf8433d8860afa20a611025621f7817b7ab16012
+PKG_GIT_HASH:=481bc77
```

Comment:
Renamed and automated. The automation uses the long hash which isn't ideal as this hash is printed in the docker CLI but it will suffice. As mentioned the hash is actually used by docker itself and will fail to compile without.

Reply:
To be honest, I prefer we do not automate it like this.  The problem is that the package makefile will be sourced quite a few times in the build process and we will need to do network io for each of them.  This can drag leg the whole build process.

I would suggest manually maintain the commit id if docker-ce does not release that much

Reply:
Reverted